### PR TITLE
Popup: update docs

### PR DESCRIPTION
**Pull request by dshovchko**

### CODE REVIEW

File: src/modules/esl-popup/README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Authors: *Alexey Stsefanovich (ala'n), Dmytro Shovchko*.
 ESLPopup is a custom element that is used as a wrapper for content that can be displayed as a pop-up GUI element.
 ESLPopup is based on ESLToggleable and works in pair with ESLTrigger that allows triggering ESLPopup instances state changes.
 
+ESLPopup can only function correctly if it is only inside the document.body. Therefore, when it connects to the DOM, it checks and if it is not in the body, it moves itself to the document.body and leaves an ESLPopupPlaceholder element in its place.
```
Comment (fshovchko) [inline src/modules/esl-popup/README.md:14]:
```suggestion
ESLPopup can only function correctly if it is only inside the document.body. Therefore, when it connects to the DOM, it checks if it's in the body, and if it's not, it moves itself to the document.body and leaves an ESLPopupPlaceholder element in its place. 
```

File: src/modules/esl-popup/README.md [hunk 2]
```diff
@@ -21,8 +23,12 @@ ESLPopup is based on ESLToggleable and works in pair with ESLTrigger that allows
   - `fit-minor` - same as fit, but position dynamically updates only on minor axes. Looks like alignment to the arrow
   - empty or unsupported value - will not be prevented from overflowing clipping boundaries, such as the viewport
   
+- `container` - target to container element ([TraversingQuery](../esl-traversing-query/README.md) selector) to define bounds of popup visibility (window by default)
```
Comment (ala-n) [inline src/modules/esl-popup/README.md:26]:
```suggestion
- `container` - defines container element ([TraversingQuery](../esl-traversing-query/README.md) selector) to determinate bounds of popup visibility (window by default)
```

**Review by ala-n — APPROVED** [review]

File: src/modules/esl-popup/README.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ Authors: *Alexey Stsefanovich (ala'n), Dmytro Shovchko*.
 ESLPopup is a custom element that is used as a wrapper for content that can be displayed as a pop-up GUI element.
 ESLPopup is based on ESLToggleable and works in pair with ESLTrigger that allows triggering ESLPopup instances state changes.
 
+ESLPopup can only function correctly if it is only inside the document.body. Therefore, when it connects to the DOM, it checks if it's in the body, and if it's not, it moves itself to the document.body and leaves an ESLPopupPlaceholder element in its place.
```
Comment (NastaLeo) [inline src/modules/esl-popup/README.md:14]: ESLPopup can only function correctly if it is inside the document.body (maybe we can remove the second only)

**Review by NastaLeo — APPROVED** [review]

File: src/modules/esl-popup/README.md (hunk 3 above)
Comment (dshovchko) [reply src/modules/esl-popup/README.md]: The second 'only' was replaced with 'placed directly'.
Thanks!

**Review by julia-murashko — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

**Pull request merged**